### DISCUSSION TITLE
turn off openMP

### DISCUSSION THREAD

**Discussion by jiandewang** [ufs-community/ufs-weather-model#2006] [Q&A]
I am working with GFDL testing one of its MOM6 PR and need to run one S2SW case under rt.sh with openMP turned off (even it's set as 1, we need to remove it from compiling and job_card). Can someone show me which scripts I need to modify ?  Thanks.

**Comment by junwang-noaa** [comment]
@jiandewang I don't think we compile MOM6 with opemp, you can see the compile options in the compile log file out. E.g. this is one of mom6 file compile line:

cd /scratch1/NCEPDEV/stmp2/Jun.Wang/FV3_RT/rt_272973/compile_s2s_intel/build_fv3_s2s_intel/MOM6-interface && /apps/oneapi/mpi/2021.5.1/bin/mpiifort  -I/scratch1/NCEPDEV/stmp2/Jun.Wang/FV3_RT/rt_272973/compile_s2s_intel/build_fv3_s2s_intel/MOM6-interface/mod -I/scratch1/NCEPDEV/nems/Jun.Wang/nems/vlab/20230817/ufs-weather-model/MOM6-interface/MOM6/config_src/memory/dynamic_nonsymmetric -I/scratch1/NCEPDEV/nems/Jun.Wang/nems/vlab/20230817/ufs-weather-model/MOM6-interface/MOM6/src/framework -I/scratch1/NCEPDEV/stmp2/Jun.Wang/FV3_RT/rt_272973/compile_s2s_intel/build_fv3_s2s_intel/stochastic_physics/mod -I/scratch1/NCEPDEV/nems/role.epic/spack-stack/spack-stack-1.4.1/envs/unified-env/install/intel/2021.5.0/fms-2023.01-ddgwd6p/include_r8 -I/scratch1/NCEPDEV/nems/role.epic/spack-stack/spack-stack-1.4.1/envs/unified-env/install/intel/2021.5.0/netcdf-fortran-4.6.0-tl53xkj/include -I/scratch1/NCEPDEV/nems/role.epic/spack-stack/spack-stack-1.4.1/envs/unified-env/install/intel/2021.5.0/netcdf-c-4.9.2-hu3jvmn/include -I/scratch1/NCEPDEV/nems/role.epic/spack-stack/spack-stack-1.4.1/envs/ufs-pio-2.5.10/install/intel/2021.5.0/esmf-8.4.2-p67qcam/include -I/scratch1/NCEPDEV/nems/role.epic/spack-stack/spack-stack-1.4.1/envs/ufs-pio-2.5.10/install/intel/2021.5.0/parallelio-2.5.10-jzyrb4d/include -**g -traceback -i4 -r8 -fno-alias -auto -safe-cray-ptr -ftz -assume byterecl -sox  -O2 -debug minimal -fp-model source -module mod** -c /scratch1/NCEPDEV/nems/Jun.Wang/nems/vlab/20230817/ufs-weather-model/MOM6-interface/MOM6/src/equation_of_state/MOM_temperature_convert.F90 -o CMakeFiles/mom6_obj.dir/MOM6/src/equation_of_state/MOM_temperature_convert.F90.o

**Reply by jiandewang** [reply]
@junwang-noaa thanks.
